.github/workflows/dependabot-auto-merge.yaml

```diff
@@ -17,10 +17,34 @@ jobs:
         uses: dependabot/fetch-metadata@c9c4182bf1b97f5224aee3906fd373f6b61b4526 # v1.6.0
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
+      - name: Check whether anyone besides dependabot has pushed
+        id: pr_author
+        run: |
+          EXTRA_AUTHORS=$(gh pr view "$PR_URL" --json commits --jq '.commits[] | .authors[] | .login' | sort | uniq | grep -v dependabot)
+          echo "Additional PR Authors: $EXTRA_AUTHORS"
+          echo "::set-output name=extra_pr_authors::$EXTRA_AUTHORS"
+          if [ -n "$EXTRA_AUTHORS" ]; then
+            echo "::set-output name=human_pushed::true"
```

Comment:
From @wagoodman offline: We should use `>> $GITHUB_OUTPUT` instead, the `::set-output` still works, but is no longer recommended.